RFCs/FS-1126-allow-lower-case-du-cases when-require-qualified-access-is specified.md

```diff
@@ -44,6 +47,26 @@ type DU = a
 type DU = | a
 
 type DU = | a
+
+[<RequireQualifiedAccess>]
+type DU =
+     | a of int
+     | B of string
+     | C
+     | ``D`` of bool
+     | ``d``
+     
+ [<RequireQualifiedAccess>]
+ type DU = ``a``
+
+ [<RequireQualifiedAccess>]
+ type DU = ``A``
```

Comment:
I don’t see the merit of these examples as they’re already valid (the bottom one, I mean). Can you add a comment as to how these relate to this RFC?

Reply:
I think you are right :) . As this is RFC is about lowercase DU I thought adding an example with Uppercase might help . But based on your comment I guess not :)